### PR TITLE
Review JavaScript1-Week1

### PR DESCRIPTION
**Pull request by zool777**

### CODE REVIEW

File: javascript/javascript1/week1/index.js [hunk 1]
```diff
@@ -12,7 +13,7 @@ dogYearFuture = 2027;
 dogYear = dogYearFuture - dogYearOfBirth;
 let shouldShowResultInDogYears = dogYear;
 
-if (shouldShowResultInDogYears!==true) {
+if (shouldShowResultInDogYears !== true) {
   console.log(
     `Your dog will be ${dogYear * 7} dog years old in ${dogYearFuture}`
   );
```
Comment (dinythomas89) [inline javascript/javascript1/week1/index.js:19]: I can't see you define the variables like yearFuture, age, dogYear. And use const for variables that don't change eg const yearOfBirth = 1987;

File: javascript/javascript1/week1/index.js [hunk 2]
```diff
@@ -12,7 +13,7 @@ dogYearFuture = 2027;
 dogYear = dogYearFuture - dogYearOfBirth;
 let shouldShowResultInDogYears = dogYear;
```
Comment (dinythomas89) [inline javascript/javascript1/week1/index.js:14]: Why you need to assisgn dogYear to shouldShowResultInDogYears?

**Review by dinythomas89 — APPROVED** [review]

File: javascript/javascript1/week1/index.js (hunk 1 above)
Comment (zool777) [reply javascript/javascript1/week1/index.js:19]: I think i forgot to assign them:
const dogYearOfBirth = 2017;
const dogYearFuture = 2027;
let dogYear = dogYearFuture - dogYearOfBirth;

File: javascript/javascript1/week1/index.js (hunk 2 above)
Comment (zool777) [reply javascript/javascript1/week1/index.js:14]: const dogYear = dogYearFuture - dogYearOfBirth;
let shouldShowResultInDogYears = dogYear;

**Pull request merged**